src/fr-webauthn/index.ts

```diff
@@ -346,8 +376,10 @@ abstract class FRWebAuthn {
   ): PublicKeyCredentialCreationOptions {
     const { pubKeyCredParams: pubKeyCredParamsString } = metadata;
     const pubKeyCredParams = parsePubKeyArray(pubKeyCredParamsString);
-    if (!pubKeyCredParams || !pubKeyCredParams.length) {
-      throw new Error('Missing pubKeyCredParams');
+    if (!pubKeyCredParams) {
+      const e = new Error('Missing pubKeyCredParams property from registration options');
+      e.name = WebAuthnOutcomeType.DataError;
+      throw e;
```

Comment:
We originally checked for both property existence and array length, but the spec says length can be zero, so I just stuck with property check.